src/malli/clj_kondo.cljc

```diff
@@ -124,10 +124,16 @@
 (defmethod accept :qualified-symbol [_ _ _ _] :symbol)
 (defmethod accept :uuid [_ _ _ _] :any) ;;??
 
-(defmethod accept :+ [_ _ [child] _] {:op :rest, :spec child})
-(defmethod accept :* [_ _ [child] _] {:op :rest, :spec child})
-(defmethod accept :? [_ _ [child] _] {:op :rest, :spec child})
-(defmethod accept :repeat [_ _ [child] _] {:op :rest, :spec child})
+(defn -seqable-or-rest [_ _ [child] {:keys [arity]}]
+  (if (= arity :varargs)
+    {:op :rest :spec child}
+    :seqable))
+
+(defmethod accept :+ [_ _  children options] (-seqable-or-rest nil nil children options))
```

Comment:
no need to pass the extra `nil` values here?

Reply:
Updated the required args.